src/test/java/org/truffleruby/tck/RubyTckTest.java

```diff
@@ -298,12 +298,13 @@ public void testWriteToObjectWithElement() throws Exception {
         return builder
                 .config(RubyLanguage.MIME_TYPE, OptionsCatalog.EXCEPTIONS_TRANSLATE_ASSERT.getName(), false)
                 .config(RubyLanguage.MIME_TYPE, OptionsCatalog.HOME.getName(), cwd)
-                .config(RubyLanguage.MIME_TYPE, OptionsCatalog.BASICOPS_INLINE.getName(), false);
+                .config(RubyLanguage.MIME_TYPE, OptionsCatalog.BASICOPS_INLINE.getName(), false)
+                .config(RubyLanguage.MIME_TYPE, OptionsCatalog.GRAAL_WARNING_UNLESS.getName(), false);
     }
 
     @Override
     public void testPropertiesInteropMessage() {
-        // Skipped as I can't get it to work with new interop CS 22-Mar-18
+        // Does KEYS and then READs each key - doesn't work because it's a Struct and so tries to #[] each method
```

Comment:
Could we have sth else than a `Struct` as the receiver for this example?
Alternatively, maybe `Struct` should be like Array and Hash and gives only the Struct members as keys, not methods.

Reply:
Yes could be a good idea - I'll make a note.